src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -362,36 +225,13 @@ private UpdateSite.Plugin extractFromLocalCheckout() throws PluginSourcesUnavail
         return new UpdateSite.Plugin(DEFAULT_SOURCE_ID, o);
     }
 
-    protected void generateHtmlReportFile() throws IOException {
-        if (!config.reportFile.exists() || !config.reportFile.isFile()) {
-            throw new FileNotFoundException(
-                    "Cannot find the XML report file: " + config.reportFile);
-        }
-
-        Source xmlSource = new StreamSource(config.reportFile);
-        try (InputStream xsltStream = getXslTransformerResource()) {
-            Source xsltSource = new StreamSource(xsltStream);
-            Result result = new StreamResult(PluginCompatReport.getHtmlFilepath(config.reportFile));
-
-            TransformerFactory factory = TransformerFactory.newInstance();
-            Transformer transformer;
-            try {
-                transformer = factory.newTransformer(xsltSource);
-                transformer.transform(xmlSource, result);
-            } catch (TransformerException e) {
-                throw new RuntimeException(e);
-            }
-        }
-    }
-
-    private static InputStream getXslTransformerResource() {
-        return PluginCompatTester.class.getResourceAsStream("resultToReport.xsl");
-    }
-
     private static File createBuildLogFile(
-            File reportFile, String pluginName, String pluginVersion, MavenCoordinates coreCoords) {
+            File workDirectory,
+            String pluginName,
+            String pluginVersion,
+            MavenCoordinates coreCoords) {
         return new File(
-                reportFile.getParentFile().getAbsolutePath()
+                workDirectory.getAbsolutePath()
```

Comment:
Minor change in behavior: this now goes in the working directory rather than in the parent directory of the XML file. These were probably the same anyway.